.github/workflows/on-release.yml

```diff
@@ -28,6 +28,9 @@ jobs:
     - name: Install dependencies ⏬
       run: npm ci
 
+    - name: Adapt base path for gh-pages :twisted_rightwards_arrows:
+      run: sed -i 's/\htmlWebpackPlugin.options.appPrefix/\/latest\//' resources/public/index.ejs
```

Comment:
Basically +1, have you tested the updated path on the `gh-pages` branch beforehand?